### DISCUSSION TITLE
SusAlert overlay is behind RuneScape window in full screen mode

### DISCUSSION THREAD

**Discussion by renegadeandy** [whs/runekit#32] [Q&A macOS]
I play RuneScape on M1 Mac and trying out rune kit for the first time.

I use fullscreen mode. When I run the SusAlert  plugin - https://github.com/Raphire/SusAlert the box loads, but it loads in a window which sits 'behind' the RuneScape window. This means I can't see the dialog when playing in fullscreen mode. On windows, this SusAlert box is overlayed on top of the RuneScape window.

Is there some configuration I have missed?

**Comment by pekempy** [comment]
I recommended trying Skillbert/Alt1-Electron, an official rewrite in electron that's cross platform

**Reply by schoentoon** [reply]
I recommend to at least read the readme of that repository.. There's currently no MacOS specific bits in that repository.. So the odds of this working at all are pretty much not existent.

**Reply by renegadeandy** [reply]
I Agree @schoentoon . Any specific thoughts to this problem I have? Is it repeatable for you?

**Reply by schoentoon** [reply]
I am not on MacOS unfortunately so I'm afraid I won't be of much help. But I suspect it has to do with how MacOS handles its full screen, making it close to (if not completely) impossible to draw things over the screen. At least, that is assuming it functions when not in full screen.
